In the c++11 standard the return value type of the ifstream::open class changed.
This makes use of the inherited ios::good member to implement the check.
Author: Davide Vanzo (Vanderbilt University)
diff -ru fsl.orig/src/melodic/meldata.cc fsl/src/melodic/meldata.cc
--- fsl.orig/src/melodic/meldata.cc	2019-09-06 15:55:18.972999005 -0500
+++ fsl/src/melodic/meldata.cc	2019-09-06 15:56:22.617000766 -0500
@@ -1015,7 +1015,7 @@
       Resels = 1.0;
       
       in.open(logger.appendDir("smoothest").c_str(), ios::in);
-      if(in>0){
+      if(in.good()){
 				for(int ctr=1; ctr<7; ctr++)
 					in >> str;
 				in.close();
